package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "undertaker",
-  "version": "0.13.1",
+  "version": "0.13.2",
```

Comment:
don't pre-bump this.  I use `npm version <major || minor || patch>` to do versioning